lib/code_corps_web/guardian_serializer.ex

```diff
@@ -10,6 +10,24 @@ defmodule CodeCorpsWeb.GuardianSerializer do
   def for_token(_), do: {:error, "Unknown resource type"}
 
   def from_token("Project:" <> id), do: {:ok, Repo.get(Project, id)}
-  def from_token("User:" <> id), do: {:ok, Repo.get(User, id)}
+  def from_token("User:" <> id) do
+    user = Repo.get(User, id)
+
+    if user do
+      name = full_name(user)
+      %Timber.Contexts.UserContext{id: user.id, email: user.email, name: name}
+      |> Timber.add_context()
+    end
+
+    {:ok, user}
+  end
   def from_token(_), do: {:error, "Unknown resource type"}
```

Comment:
This is sort of adding a side-effect to the serializer.

I would be more comfortable specifying a separate, more explicit plug which adds this context.

Something like the combination of `CodeCorpsWeb.Plug.Segment` and `CodeCorps.Analytics.SegmentPlugTracker`, except in a single module.

Reply:
In the interest of getting this merged, could we open an issue to circle back and address it? This, for me, is lower priority than other things on the totem pole.

Reply:
#932